### PR TITLE
My homework2

### PR DESCRIPTION
**Pull request by Bondarchuk97Ilja**
Только там блоки закоменчены - /* */. Забыл убрать перед тем как запушить, но не думаю что это проблема)

### CODE REVIEW

File: homework2/homework2.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/*1-ый блок
```
Comment (moroz112) [inline homework2/homework2.js:1]: Why is everything commented out?

File: homework2/homework2.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+5. +'10' + 1 = 11 - унарное преобразование
+6. undefined + 2 = NaN - из-за undefined
+7. null + 5 = 5 - null преобразуется в  0
+8. true + undefined = NaN - true = 1, но undefined NaN
+*/
+/*
+//3-ий блок
+//1. Если переменная равна “hidden”, присвоить ей значение “visible”, иначе - “hidden”.
+let a = 'hidden';
+if (a =='hidden'){
```
Comment (moroz112) [inline homework2/homework2.js:29]: It's ok. But using strict comparison `===` is better

File: homework2/homework2.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+alert(  car["needRepair"]);
+
+//4.Дан объект let item = { name: 'Intel core i7', price: '100$', discount: '15%' }.
+// Написать условие если у item есть поле discount и там есть значение то в объекте item создать
+// поле priceWithDiscount и записать туда цену с учетом скидки и вывести ее в консоль, обратите
+// внимание что поля discount и price это строки и вам из них нужно получить числа чтобы выполнить
+// расчет. иначе если поля discount нет то вывести просто поле price в консоль.
+let item = { name: 'Intel core i7', price: '100$', discount: '15%'};
+if (item.discount){
+    item["priceWithDiscount"]=parseInt(item["price"])-parseInt(item["price"])*(parseInt(item["discount"])/100)
```
Comment (moroz112) [inline homework2/homework2.js:70]: would be simpler if you'd use `let productPrice = parseInt(product.price); if(productPrice >= min) .....` In this case you'd have done operation parseInt once not twice

File: homework2/homework2.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+// };
+// let min = 10; // минимальная цена
+// let max = 20; // максимальная цена
+// Написать условие если цена товара больше или равна минимальной цене и меньше или равна максимальной
+// цене то вывести в консоль название этого товара, иначе вывести в консоль что товаров не найдено.
+
+let product = {name:"Яблоко",price:"10$"};
+let min = 10;
+let max = 20;
+if (parseInt(product["price"])>=min && parseInt(product["price"])<=max){
```
Comment (moroz112) [inline homework2/homework2.js:88]: same with `parseInt`

**Review by moroz112 — APPROVED** [review]
Really good job. Except everything is commented out

File: homework2/homework2.js (hunk 1 above)
Comment (Bondarchuk97Ilja) [reply homework2/homework2.js:1]: i forgot uncomment before pushing. Next time everything will be ok.

**Pull request merged**